.empty-mod/go.mod

```diff
@@ -0,0 +1,11 @@
+// module empty-mod is an empty module that's used to help with containerd
```

Comment:
Nit: Don't we need the copyright header here?

Reply:
I looked at the main go.mod, which didn't have it. I also doubt if people would copy it (likely not useful to copy another project's go.mod), let alone if it's technically copyright'able (as its mostly mechanical, no real creativity)

That said, we can have a look at adding the header to other files, such as the github action workflows etc